hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```
@@ -1242,7 +1246,12 @@ public AbfsRestOperation getPathStatus(final String path,
           && isImplicitCheckRequired && isNonEmptyDirectory(path, tracingContext)) {
         // Implicit path found.
         // Create a marker blob at this path.
-        this.createMarkerAtPath(path, null, contextEncryptionAdapter, tracingContext);
+        try {
```

For confirmation- we want to swallow exceptions only for metadata related operations and allow permission exceptions for all write ones (createDirectory, createFile)?

---

marker creation is best effort for us, we don't want to fail the actual operation called if marker creation fails, so only for marker creation we are swallowing exception and not blocking the actual call